src/tv2-common/videoSwitchers/TriCaster.ts

```diff
@@ -175,17 +191,79 @@ export class TriCaster extends VideoSwitcherImpl {
 		)
 	}
 
-	public getDveTimelineObjects(_properties: DveProps): TSR.TSRTimelineObj[] {
-		throw new Error('Method not implemented.')
-	}
-
 	public updateUnpopulatedDveBoxes(
 		_timelineObject: TSR.TSRTimelineObj,
 		_input: number | SpecialInput
 	): TSR.TSRTimelineObj {
 		throw new Error('Method not implemented.')
 	}
 
+	public getDveTimelineObjects(dveProps: DveProps): TSR.TSRTimelineObj[] {
+		return [
+			literal<TSR.TimelineObjTriCasterME>({
+				id: '',
+				enable: dveProps.enable ?? { start: 0 },
+				layer: TRICASTER_LAYER_PREFIX + SwitcherDveLLayer.DveBoxes,
+				priority: 1,
```

Comment:
This seems more consistent with the rest
```suggestion
				...this.getBaseProperties(dveProps, SwitcherDveLLayer.DveBoxes),
```